CONTRIBUTING_ORGANIZATIONS.md

```diff
@@ -0,0 +1,46 @@
+# Contributing Organizations
+
+A **Contributing Organization** is an entity that actively supports the Kairos project by providing **one or more maintainers** to help with its development, governance, and community efforts. These organizations play a critical role in ensuring the project's sustainability and growth, aligning with the Cloud Native Computing Foundation’s (CNCF) **open governance model**. 
+
+As Kairos is a CNCF project, its development is driven by a diverse set of contributors rather than a single vendor. Contributing organizations ensure that the project remains **vendor-neutral**, community-driven, and evolves in a way that benefits the broader ecosystem. 
+
+| Company Name  | GitHub Organization                              | Website                       |
+|---------------|--------------------------------------------------|-------------------------------|
+| Spectro Cloud | [@Spectrocloud](https://github.com/spectrocloud) | https://www.spectrocloud.com/ |
+
+## Joining As a Contributing Organization
+
+### On the Road to Incubation
+
+Kairos is currently a **CNCF Sandbox Project**, meaning it is in the early stages of adoption but has strong potential to become a key technology in the cloud-native ecosystem. One of our key goals is to progress from **Sandbox to Incubation**, which requires demonstrating **diverse organizational backing** and a healthy contributor base. To achieve this, we need more than one contributing organization actively supporting the project. Your participation as a **contributing organization** would be instrumental in helping us reach this milestone and solidify Kairos as a leading cloud-native solution.
+
+### Roadmap
+
+Kairos follows a transparent and community-driven [**roadmap**](https://github.com/orgs/kairos-io/projects/2), which outlines planned features, priorities, and ongoing efforts. Organizations contributing maintainers are encouraged to align their contributions with this roadmap to ensure consistency and project-wide progress. The latest roadmap is publicly available at:
```

Comment:
`available at:` ? missing something

Reply:
whoops, silly, I moved it to the first part of the sentence and forgot to change it. Great catch!